src/vertex_pulling/queue.rs

```diff
@@ -33,9 +35,12 @@ pub(crate) fn queue_cuboids(
                         cuboids_pipelines.pipeline_id
                     };
                     opaque_phase.add(Opaque3d {
+                        // TODO: https://github.com/bevyengine/bevy/pull/11671/files
+                        asset_id: AssetId::invalid(),
+                        // asset_id: CUBE_INDICES_HANDLE.into(),
+                        // distance: inverse_view_row_2.dot(entry.position.extend(1.0)),
                         pipeline,
                         entity,
-                        distance: inverse_view_row_2.dot(entry.position.extend(1.0)),
```

Comment:
@Elabajaba since you wrote https://github.com/bevyengine/bevy/pull/11671, how is an app like this that uses a simple index buffer and "generates"/computes vertex positions "on the fly" in the vertex shader supposed to provide an `AssetId<Mesh>`? Looks like we can't get the original sorting based on `distance`.

Reply:
I'm not sure tbh. I didn't realize anyone was doing vertex pulling with bevy's standard opaque3d pipeline.

I opened a PR on your fork (https://github.com/MarijnS95/bevy_aabb_instancing/pull/1) that re-implements bevy's old `Opaque3d` pass as a custom pass to get this working, but I'm not sure if it's the best solution.

Reply:
@Elabajaba that sounds unfortunate.

Thanks for the ping. Looks like creating forks via the `gh` CLI doesn't automatically set myself as watching my own repo, so I didn't receive any other notification for you opening a PR there.

I'll take a closer look later but already see some drive-by fixes that you ought to better submit separately :)